### PR TITLE
 OCM-6029 | feat: allow to edit component routes of ingress

### PR DESCRIPTION
**Pull request by gdbranco**
[OCM-6029](https://issues.redhat.com//browse/OCM-6029)

### CODE REVIEW

File: cmd/edit/ingress/cmd.go [hunk 1]
```diff
@@ -423,41 +392,70 @@ func run(cmd *cobra.Command, argv []string) {
...
+			componentRoutes, err = parseComponentRoutes(args.componentRoutes)
 			if err != nil {
-				r.Reporter.Errorf("Expected a valid Cluster Routes TLS Secret Reference: %s", err)
+				r.Reporter.Errorf("An error occurred whilst parsing the supplied component routes: %s", err)
 				os.Exit(1)
 			}
-			clusterRoutesTlsSecretRef = &clusterRoutesTlsSecretRefArg
+		} else if interactive.Enabled() && !ocm.IsHyperShiftCluster(cluster) {
+			componentRoutes = map[string]*cmv1.ComponentRouteBuilder{}
+			if confirm.Prompt(false, "Would you like to edit the component routes?") {
```
Comment (oriAdler) [inline cmd/edit/ingress/cmd.go:430]: Is there a way to simplify the `if else` conditions? is it worth extracting some of the logic to new functions?

Comment (gdbranco) [reply cmd/edit/ingress/cmd.go:427]: hmm, can only really setup a variable for it. Can't think of a different way at the moment :thinking:

**Comment by oriAdler** [comment]
@gdbranco it seems like the `ocm-sdk-go` version on the master branch was updated to `v0.1.409` and this PR needs a rebase.

**Pull request merged**